Makefile

```diff
@@ -75,8 +75,7 @@ endif
 
 ifeq ($(PLATFORM),sdl)
 # the dir containing /bin, /include, /lib
-#SDL_DIR := $(error Add SDL directory)
-SDL_DIR := /mnt/c/Code/Libraries/SDL2_Root/_32
+SDL_DIR := $(error Please add the directory path to SDL2)
```

Comment:
for linux, you can use `pkg-config --cflags --libs sdl2` for the required flags

Reply:
You could also use `-I/usr/include/SDL2` for linux (instead of the mnt path you had here)

Reply:
I feel like it's better to either directly ship with it and build it or have people add their own paths.

- not everyone has pkg-config installed
- it's not guaranteed that they would install it to `/usr/include/SDL2`

Reply:
Or maybe add a standard command that looks for it in the `/usr/` dir and display the error if it can't find it?

Reply:
Can figure it out later

Reply:
> for linux, you can use `pkg-config --cflags --libs sdl2` for the required flags

iirc you can also use `sdl2-config --cflags --libs`